debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java

```diff
@@ -1082,6 +1101,17 @@ public String getConnectorName() {
         return Module.name();
     }
 
+    @Override
```

Comment:
@govi20 This and following bethod could be also pulled-up togetr with the relevant attributes.

Reply:
Done